packages/ember-runtime/tests/computed/reduce_computed_macros_test.js

```diff
@@ -614,40 +615,45 @@ QUnit.test("it has set-intersection semantics", function() {
 });
 
 
-QUnit.module('computedSetDiff', {
-  setup: function() {
-    run(function() {
-      obj = EmberObject.createWithMixins({
-        array: Ember.A([1,2,3,4,5,6,7]),
-        array2: Ember.A([3,4,5,10]),
-        diff: computedSetDiff('array', 'array2')
-      });
-    });
-  },
-  teardown: function() {
-    run(function() {
-      obj.destroy();
-    });
-  }
-});
+forEach.call([['difference', computedDifference], ['setDiff', computedSetDiff]], function (tuple) {
```

Comment:
One thing I was unsure about here – there's [another test](https://github.com/imkmf/ember.js/blob/alias-computed-setdiff-to-difference/packages/ember-runtime/tests/computed/reduce_computed_macros_test.js#L660) related to `setDiff` below this. I re-ordered the array inside of `forEach` so that the additional test would end up as part of the `setDiff` module. It's not a super intuitive order for the array, however, because `difference` is not the primary method being tested. Should I update that?